docs/mobile/sdk-components-reference.mdx

```diff
@@ -130,7 +130,7 @@ Check out this example of a [component with a view model](https://github.com/geo
 
 ## Configuration Models
 
-Every component is bound to specific [**item type**](configuration-app-config-reference.mdx#app-items), and each item type is bound to a class [registered as an `AppItem` with autofac](sdk-dependency-injection.mdx#appitem). This class instantiates itself with values from the app config in it's constructor, acting as a configuration model. The component class then consumes this configuration model, and then passes the appropriate values to the view and view model.
+Every component is bound to specific [**item type**](configuration-app-config-reference.mdx#app-items), and each item type is bound to a class [registered as an `AppItem` with autofac](sdk-dependency-injection.mdx#appitem). This class instantiates itself with values from the app config in its constructor, acting as a configuration model. The component class then consumes this configuration model, and then passes the appropriate values to the view and view model.
```

Comment:
"The component class **then** consumes this configuration model, and then passes the appropriate values to the view and view model."

I'd remove the first instance of 'then'.

Reply:
I was just fixing the spelling of "its" here. Strictly speaking that was out of scope and I'd rather not get sidetracked fixing more.